lib/src/main/java/graphql/nadel/NextgenEngine.kt

```diff
@@ -296,12 +296,7 @@ class NextgenEngine @JvmOverloads constructor(
             )
         }
 
-        return serviceExecResult.copy(
-            data = serviceExecResult.data.let { data ->
-                data?.takeIf { transformedQuery.resultKey in data }
-                    ?: mutableMapOf(transformedQuery.resultKey to null)
```

Comment:
This code was here to ensure that the result always has the top level field.

This has been moved to `NadelResultMerger` and now handles namespaced fields too.